corehq/apps/locations/models.py

```diff
@@ -228,7 +228,7 @@ class SQLLocation(SyncSQLToCouchMixin, MPTTModel):
     location_type = models.ForeignKey(LocationType)
     site_code = models.CharField(max_length=255)
     external_id = models.CharField(max_length=255, null=True)
-    metadata = json_field.JSONField(default={})
+    metadata = jsonfield.JSONField(default={})
```

Comment:
I think if we migrate to django's jsonfield this would break as well https://docs.djangoproject.com/en/1.9/ref/contrib/postgres/fields/#querying-jsonfield